### PR TITLE
Improving usability (better configuration & intro to docker plugin)

### PR DESCRIPTION
**Pull request by asnir**
1. Making the code more generic (e.g. The program will get the zookeeper value as arg )
2. Update kafka.version from 0.9.0.1-cp1 to 0.9.0.1 (pom.xml)
3. Adding docker-maven-plugin

### CODE REVIEW

**Comment by asnir** [comment]
@miguno

**Comment by miguno** [comment]
@asnir:  This PR introduces quite a few changes that are not related to each other (e.g. the introduction of some docker functionality vs. more meaningful variable names vs. better CLI parsing).  Some of these changes are easier/quicker to accept and review than others, so if it works for you I'd suggest to split the PR into separate ones (which can then also have more descriptive PR titles rather than a generic "code refactoring").

File: kafka-clients/consumer/pom.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
     <properties>
         <java.version>1.7</java.version>
-        <kafka.version>0.9.0.1-cp1</kafka.version>
```
Comment (miguno) [inline kafka-clients/consumer/pom.xml]: For now we want to keep the version at `0.9.0.1-cp1`.

File: kafka-clients/consumer/pom.xml [hunk 2]
```diff
@@ -88,6 +88,21 @@
                     </licenseMapping>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
```
Comment (miguno) [inline kafka-clients/consumer/pom.xml]: Why the shading?

**Comment by miguno** [comment]
Closed because of inactivity. Feel free to reopen if needed (and see review comments above).

**Pull request closed without merging**